src/test/groovy/org/prebid/server/functional/model/response/amp/AmpResponse.groovy

```diff
@@ -1,11 +1,7 @@
 package org.prebid.server.functional.model.response.amp
 
-import org.prebid.server.functional.model.response.BidderError
-import org.prebid.server.functional.model.response.Debug
-
 class AmpResponse {
 
-    Map<String, String> targeting
-    Debug debug
-    Map<String, List<BidderError>> errors
+    AmpResponseBody responseBody
```

Comment:
Bad idea. You will need headers for only a couple of tests, but you are complicating model for everything else. Better add a method to a PBS service to return raw response body and headers.

Reply:
You can leave this class as a wrapper for that service method to return, but it should not replace existing interactions. But the name will need to change.